### PR TITLE
Updated the function to retrive the tenantId

### PR DESCRIPTION
**Pull request by Vidya2606**
# Description

Addressing a comment which was missed in the previous PR
https://github.com/Azure/draft/pull/258
Screenshot 2024-03-27 at 9 48 33 AM

Fixes # (issue)
Feature # (details)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration. Is it a  breaking change which will impact consuming tool(s).

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by bfoley13 — APPROVED** [review]

File: pkg/providers/azure.go [hunk 1]
```diff
@@ -193,7 +193,7 @@ func (sc *SetUpCmd) getTenantId(ctx context.Context) error {
 	if len(tenants) > 1 {
 		return errors.New("multiple tenants found")
 	}
-	sc.tenantId = fmt.Sprint(&tenants[0])
+	sc.tenantId = *tenants[0].TenantID
```
Comment (bfoley13) [inline pkg/providers/azure.go:196]: I know the sdk's can be difficult to test, but might be something we look into options for

**Comment by OliverMKing** [comment]
can you attach a screenshot of this working?

**Review by davidgamero — APPROVED** [review]
Lgtm

File: pkg/providers/azure.go (hunk 1 above)
Comment (davidgamero) [reply pkg/providers/azure.go:196]: Agreed we can at least mock the calls to the api

**Pull request merged**